automation-swiftui/AutomationAssessment/ContentView.swift

```diff
@@ -55,39 +49,38 @@ struct ContentView: View {
                 .font(.subheadline)
         }
     }
-
+    
     var headerView: some View {
-        VStack {
+        VStack(alignment: .leading) {
             titleView
             placeholderImage
             subtitleView
         }
     }
-
+    
     var tagSelector: some View {
         HStack {
+            Picker("Topic", selection: $selectedTopic) {
```

Comment:
As discussed on slack, the screen reader goes to 'go to politics' before the picker - I'd swap these around, even for non visually impaired candidates it seems odd that you pick from the picker on the right and then go to it on the left when we read left to right, which makes the sample app confusing.